WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -349,15 +349,31 @@ class MySiteViewModel @Inject constructor(
         }
     }
 
-    private fun getPositionOfQuickStartItem(siteItems: Map<MySiteTabType, List<MySiteCardAndItem>>) =
+    private fun getPositionOfQuickStartItem(
+        siteItems: Map<MySiteTabType, List<MySiteCardAndItem>>,
+        activeTask: QuickStartTask
+    ) =
             if (isMySiteTabsEnabled) {
```

Comment:
Nit Pick: The `if` can move up to the previous line and the fun can be reformatted, that will decrease the indent of the `.indexOfFirst` below

Reply:
@zwarm Yeah, good point.  I have moved up the `if` and formatted the fun. Please find the changes in 09e34c3. 👍🏼